index.html

```diff
@@ -1055,105 +1038,87 @@ <h4>Attributes</h4>
       <div>
         <pre class="idl">
   dictionary RTCIceCandidatePairEventInit : EventInit {
-    required RTCIceCandidate local;
-    required RTCIceCandidate remote;
+    required RTCIceCandidatePair candidatePair;
```

Comment:
Was this change discussed?
It looks ok to me but this reduces the ability of creating event to candidate pairs created by the UA